### ISSUE TITLE
Quite expected error

### ISSUE THREAD

**Issue by s-zanella** [FStarLang/FStar#449]
```
$ ls foo.fst
ls: cannot access foo.fst: No such file or directory
$ fstar.exe foo.fst
Unexpected error; please file a bug report, ideally with a minimized version of the source program that triggered the error.
Could not find file 'C:\cygwin\home\santiago\FStar\foo.fst'.
```

**Comment by beurdouche** [comment]
```
$ ls foo.fst
ls: foo.fst: No such file or directory
$ fstar.exe foo.fst
Error: Unable to open file: foo.fst
```

**Comment by s-zanella** [comment]
I still have this error with the F# build. The OCaml build behaves as expected (as in your post).

**Comment by beurdouche** [comment]
Fixed by d60bd4c
